omfhint.cpp

```diff
@@ -72,7 +75,10 @@ OMFHintFilter::ingest(vector<Reading *> *readings, vector<Reading *>& out)
 
 					if (std::regex_match (name, item.first))
 					{
-						DatapointValue value(item.second);
+						std::string hintsJSON = item.second;
+						if (m_macro_dpName.size())
```

Comment:
use empty() instead of size() if aim is to check non-empty

Reply:
fixed